source/component/PasDoc_Gen.pas

```diff
@@ -1397,6 +1414,16 @@ procedure TDocGenerator.HandleLinkTag(
   ReplaceStr := SearchLink(LinkTarget, FCurrentItem, LinkDisplay, true);
 end;
 
+procedure TDocGenerator.HandleUrlTag(
+  ThisTag: TTag; var ThisTagData: TObject;
```

Comment:
TObject doesn't need to be declared as `var`

Reply:
That's given by the definition of `TTagExecuteEvent`